serverless.yml

```diff
@@ -25,7 +25,7 @@ custom:
   # This is the 'variable' for the customDomain.basePath value, based on the stage.
   # Run as `sls deploy --stage <stage_name>`
   stage:
-    prod: prod
+    prod: projects
```

Comment:
This will affect the table name (see line 18). Ideally we should have a consistent way of naming the tables and urls, but we don't yet. So let's stay safe and not include this change yet, at least until we've decided how we want those naming patterns to work.